R/sircovid.R

```diff
@@ -152,10 +152,13 @@ run_particle_filter <- function(data,
   
   #set up compare for observation likelihood
   compare_func <- sircovid_model$compare_model(model = model_func, pars_obs = obs_params, data = data)
+  
+  seeding_func <- sircovid_model$seeding_model(model = model, data = data, pars_seeding = NULL)
```

Comment:
I think it makes sense for `pars_seeding` to be set in the initial model call, unless this is likely to be changed between different calls to the same model

Reply:
Have put pars_seeding as an argument to run_particle_filter now. Don't think we need it as an argument in beta_date_particle_filter because I think fixed seeding is inherent to that (so just need to input pars_seeding = NULL in run_particle_filter).